perfkitbenchmarker/publisher.py

```diff
@@ -554,7 +554,11 @@ def AddSamples(self, samples, benchmark, benchmark_spec):
       sample['product_name'] = FLAGS.product_name
       sample['official'] = FLAGS.official
       sample['owner'] = FLAGS.owner
-      sample['timestamp'] = time.time()
+      if 'timestamp' in sample['metadata']:
```

Comment:
@cmccoy I just uploaded a new version that adds the timestamp to the Sample class.